ao_purchase_auto_close/tests/test_ao_purchase_auto_close.py

```diff
@@ -41,7 +41,6 @@ def setUpClass(cls):
     def _do_picking(self, picking, date):
         """Do picking with only one move on the given date."""
         picking.action_confirm()
-        picking.force_assign()
```

Comment:
force_assign() method change in v12 to action_assign(), don't remove it.

Reply:
You're right